src/hooks/useAccount.tsx

```diff
@@ -54,7 +54,7 @@ export const useAccount = () => {
     setWalletModules,
     setPrimaryWallet,
     provider,
-    account,
+    account : account ? account : readOnlyWallet,
```

Comment:
This should not required and is handled in the useMemo hook for account variable